crates/biome_tailwind_parser/tests/tailwind_specs/error/incomplete-arbirary-value-2.txt.snap

```diff
@@ -0,0 +1,111 @@
+---
+source: crates/biome_tailwind_parser/tests/spec_test.rs
+expression: snapshot
+---
+## Input
+
+```text
+text-[ text-sm bg-red-500
+
+```
+
+
+## AST
+
+```
+TwRoot {
+    bom_token: missing (optional),
+    candidates: TwCandidateList [
+        TwCandidate {
+            variants: TwVariantList [],
+            candidate: TwBogusCandidate {
+                items: [
+                    TW_BASE@0..4 "text" [] [],
+                    DASH@4..5 "-" [] [],
+                    L_BRACKET@5..7 "[" [] [Whitespace(" ")],
+                ],
+            },
+            excl_token: missing (optional),
+        },
+        WHITESPACE@7..7 "" [] [],
```

Comment:
It seems we're emitting a whitespace with zero width, which seems incorrect. We're probably attaching it to the previous token as trivia